department-of-veterans-affairs/caseflow
# Fixed focus indicator on schedule hearings table

Resolves [CASEFLOW-1670](https://vajira.max.gov/browse/CASEFLOW-1670)

### Description

Updated the table to use a fixed layout so that the table does not overflow its parent which was causing the focus border to not fully enclose the table.

### Acceptance Criteria
- [ ] Veterans Waiting tables does not extend outside the focus indicator

### Testing Plan

1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Navigate to the hearing schedule: http://localhost:3000/hearings/schedule
3. Click the “Schedule Veteran” button at the top left
4. Select the St. Petersburg, FL RO from the dropdown
5. Click the “AMA Veterans Waiting” tab 
6. Tab to the table so that it is in focus and ensure that the focus border encompasses the whole table
7. Open the browser console and enter into responsive mode to test screen sizes down to 1280px width and ensure that the focus indicator still encompasses the full table


### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1396" alt="Screen Shot 2021-06-28 at 12 16 17 PM" src="https://user-images.githubusercontent.com/61989526/123691596-aab20c80-d80a-11eb-84bd-0e533e76991d.png">|<img width="1394" alt="Screen Shot 2021-06-28 at 12 15 05 PM" src="https://user-images.githubusercontent.com/61989526/123691602-aede2a00-d80a-11eb-8e8d-89895bf979ff.png">
